cmd/nfd-topology-updater/main_test.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
```

Comment:
Agreed, I thought so too, but when I noticed https://github.com/kubernetes-sigs/node-feature-discovery/blob/master/cmd/nfd-master/main_test.go and https://github.com/kubernetes-sigs/node-feature-discovery/blob/master/cmd/nfd-worker/main_test.go and assumed that we'd be expected to do the same in case of nfd-topology-updater